packages/core/src/browser/context-menu-renderer.ts

```diff
@@ -29,6 +29,13 @@ export function toAnchor(anchor: HTMLElement | Coordinate): Anchor {
     return anchor instanceof HTMLElement ? { x: anchor.offsetLeft, y: anchor.offsetTop } : anchor;
 }
 
+export function isAnchor(arg?: Anchor): boolean {
+    if (arg && 'x' in arg && 'y' in arg && Object.keys(arg).length === 2) {
+        return true;
+    }
+    return false;
+}
```

Comment:
Is this correct and sufficient? The `Anchor` type also includes `MouseEvent`s (l. 26).

```typescript
export interface Coordinate { x: number; y: number; }
export const Coordinate = Symbol('Coordinate');
export type Anchor = MouseEvent | Coordinate;
```

Reply:
I changed the type to unknown.